pxteditor/experiments.ts

```diff
@@ -3,7 +3,8 @@ namespace pxt.editor.experiments {
         id: string; // == field in apptheme also assumes image at /static/experiments/ID.png
         name: string;
         description: string;
-        feedbackUrl: string; // allows user to put feedback
+        feedbackUrl?: string; // allows user to put feedback
+        disableOffline?: boolean;
```

Comment:
The double negative is a little confusing... can this just be called enableOnline?

Reply:
Ah good point, thanks! I did one more minor change as well to remove a cast, as casting actually removes the type checking for the experiments there (instead of telling compiler to confirm they are Experiments, it tells the compiler that you confirm they are Experiments)